lib/keystore.js

```diff
@@ -57,17 +60,22 @@ function nacl_decodeHex(msgHex) {
 }
 
 
-var KeyStore = function(mnemonic, pwDerivedKey, hdPathString) {
-
-  this.defaultHdPathString = "m/0'/0'/0'";
+var KeyStore = function(mnemonic, pwDerivedKey, hdPathString, salt) {
```

Comment:
I think we should probably leave the old constructor as it is, but add a warning that it will soon be deprecated and please use the new constructor.